scopes/workspace/install/install.main.runtime.ts

```diff
@@ -404,6 +462,18 @@ export class InstallMain {
   }
 }
 
+interface AllWorkspaceManifests {
+  componentsManifests: Record<string, ProjectManifest>;
+  workspaceManifest: ProjectManifest;
+}
+
+function isManifestsEqual(prev: AllWorkspaceManifests, next: AllWorkspaceManifests): boolean {
+  return (
+    isEqual(prev.componentsManifests, next.componentsManifests) &&
+    JSON.stringify(prev.workspaceManifest) === JSON.stringify(next.workspaceManifest)
```

Comment:
why different types of compare?
also, I wonder if the stringify compare might be affected here by the order of the keys which might be different maybe between iterations?

Reply:
Changed to isEqual in both cases. isEqual doesn't care about the order of keys.